app/scripts/modules/netflix/fastProperties/view/fastPropertyPodTable.component.ts

```diff
@@ -0,0 +1,46 @@
+import { module, IComponentController, IComponentOptions} from 'angular';
+import { IStateService } from 'angular-ui-router';
+
+export class FastPropertyPodTableController implements IComponentController {
+
+  static get $inject() {
+    return [
+      '$state'
+    ];
+  }
+
+  constructor(private $state: IStateService) {}
+
+  public showPropertyDetails = (propertyId: string) => {
```

Comment:
we should try to avoid creating function handles like this in TS classes as it kind of defeats the purpose of having classes in general.